controlpanel/api/models/app.py

```diff
@@ -51,16 +51,23 @@ def customers(self):
     def add_customers(self, emails):
         emails = list(filter(None, emails))
         if emails:
-            auth0.AuthorizationAPI().add_group_members(
-                group_name=self.slug,
-                emails=emails,
-                user_options={"connection": "email"},
-            )
+            try:
+                auth0.AuthorizationAPI().add_group_members(
+                    group_name=self.slug,
+                    emails=emails,
+                    user_options={"connection": "email"},
+                )
+            except auth0.Auth0Error as e:
+                raise AddCustomerError(e)
 
     def delete_customers(self, user_ids):
-        auth0.AuthorizationAPI().delete_group_members(
-            group_name=self.slug, user_ids=user_ids
-        )
+        try:
+            auth0.AuthorizationAPI().delete_group_members(
+                group_name=self.slug,
+                user_ids=user_ids,
+            )
+        except auth0.Auth0Error as e:
+            raise DeleteCustomerError(e)
```

Comment:
same here, we could use `raise DeleteCustomerError  from e` so that the exception keeps track of the cause of the error.